thefuck/corrector.py

```diff
@@ -24,7 +24,16 @@ def get_categories(command):
     """
     CATEGORIES = {'git': {'git'},
                   'brew': {'brew'},
-                  'django': {'manage.py', 'migrate'}}
+                  'django': {'manage.py', 'migrate'},
+                  'grep': {'grep'},
+                  'grep': {'egrep'},
+                  'cargo': {'cargo'},
+                  'maven': {'mvn'},
+                  'rm': {'rm'},
+                  'pacman': {'pacman'},
```

Comment:
the rule `pacman.py` needs to be included even if the command doesn't have `pacman` in it. The `match` function only looks for `not found` in the stderr.